common/app/model/dotcomrendering/pageElements/CalloutExtraction.scala

```diff
@@ -167,6 +173,41 @@ object CalloutExtraction {
     }
   }
 
+  private def campaignJsObjectToCalloutBlockElementV2(
+      campaign: JsObject,
+      isNonCollapsible: Option[Boolean],
+      calloutsUrl: Option[String],
+  ): Option[CalloutBlockElementV2] = {
+    for {
+      id <- (campaign \ "id").asOpt[String]
+      activeFrom <- (campaign \ "activeFrom").asOpt[Long]
+      displayOnSensitive <- (campaign \ "displayOnSensitive").asOpt[Boolean]
+      formId <- (campaign \ "fields" \ "formId").asOpt[Int]
+      title <- (campaign \ "fields" \ "callout").asOpt[String]
+      description <- (campaign \ "fields" \ "description").asOpt[String]
+      tagName <- (campaign \ "fields" \ "tagName").asOpt[String]
+      formFields1 <- (campaign \ "fields" \ "formFields").asOpt[JsArray]
+    } yield {
+      val formFields2 = formFields1.value
+        .flatMap(formFieldItemToCalloutFormField(_))
```

Comment:
Just a suggestion. Take it or leave it 🙂 

```suggestion
        .flatMap(formFieldItemToCalloutFormField)
```

Reply:
Great suggestion, thanks :)